custom_components/localtuya/light.py

```diff
@@ -185,7 +185,7 @@ def is_on(self):
     @property
     def brightness(self):
         """Return the brightness of the light."""
-        if self.is_color_mode or self.is_white_mode:
+        if (self.is_color_mode or self.is_white_mode) and self._brightness:
```

Comment:
Brightness can legitimately be 0, can't it? (ie: this would be false)

Would it be better to check if this is a None or similar value?

Reply:
I _think_ 1 might be the lowest legitimate value you can set through the app, but it's a good point & I'll update it. (Been meaning to come back to this all week, but kept forgetting to borrow the offending device out the room _before_ the child went to sleep - dare not wake them for it!)

Reply:
Updated after doing a local test